tests/e2e/common-actions/workbench-actions.ts

```diff
@@ -0,0 +1,16 @@
+import {t} from 'testcafe';
+import { WorkbenchPage } from '../pageObjects';
+
+const workbenchPage = new WorkbenchPage();
+
+export class WorkbenchActions {
+    /*
+        Verify after client list command columns are visible
+    */
+    async verifyClientListColumnsAreVisible(): Promise<void> {
```

Comment:
I think it is better to test it with loop and some dynamic selector.
e.g.
const columns = ['id', 'addr', 'name', 'user'];

for (const column of columns) {
let columnSelector = Selector(`[data-test-subj^=tableHeaderCell_${column}_]`);
await t.expect(columnSelector.visible).ok(`${column} column is not visible`);
}

does it make sense?

Reply:
so you will be able to test any number of columns described in "columns" array